extensions/wikia/UserActivity/UserActivityController.class.php

```diff
@@ -58,16 +54,16 @@ public function index() {
 	protected function formatItems( $items ) {
 		$flattened = [];
 		foreach ( $items as $sortKey => $contribItem ) {
-			$lastEditTS = wfTimestamp( TS_MW, $contribItem[ 'last_edit' ] );
+			$lastEditTS = wfTimestamp( TS_MW, $contribItem[ 'lastedit' ] );
+			unset($contribItem['lastedit']);
```

Comment:
I'm trying to update these to our naming conventions but I see what you mean.  I've updated to create a new array rather than alter the key slightly.